nuget/NativeNuget.nuspec

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<!-- ORT Extensions nuspec -->
+<package>
+	<metadata>
+	<id>Microsoft.ML.OnnxRuntime.Extensions</id>
+	<version>0.8.0</version>
+	<authors>Microsoft</authors>
+	<owners>Microsoft</owners>
+	<description>ONNX Runtime Extensions NuGet Package</description>
+	<!-- TODO: Update release notes prior to 0.8.0 release with new features here -->
+	<releaseNotes>
+		General
+		1. New custom operators: RobertaTokenizer, ClipTokenizer, EncodeImage, DecodeImage
+		2. ORT custom operator C++ stub generation tool
+		3. Operator implementation and documentation improved.
+		4. Python (3.7 - 3.10) and ORT (1.10 above) compatible.
+
+		Mobile
+		1. Android package: Maven
+		2. iOS package: CocoaPods
+		3. PrePostProcessor tool for mobile model
+		4. Super-resolution model pre- / post- processing end-to-end examples
+	</releaseNotes>
+	<copyright>© Microsoft Corporation. All rights reserved.</copyright>
+	<tags>ONNX ONNXRuntime AI Machine Learning</tags>
+	<icon>ORT_icon_for_light_bg.png</icon>
+	<license type="file">LICENSE.txt</license>
+	<projectUrl>https://github.com/Microsoft/onnxruntime-extensions</projectUrl>
+	<repository type="git" url="https://github.com/Microsoft/onnxruntime-extensions.git" commit="98a303e639abff092f77295d39a31b5411d4f110" />
+	</metadata>
+	<!-- assemble files in the /nuget directory and run nuget pack from -->
+	<files>
+		<file src="..\LICENSE.txt" target="LICENSE.txt" />
+		<file src="..\includes\*.h" target="build\native\include" />
+		<file src="..\includes\*.hpp" target="build\native\include" />
+		<!-- Windows, Linux, macOS via netstandard -->
+		<file src="..\build\Windows\Debug\bin\Debug\ortextensions.dll" target="runtimes\win-x64\native" />
```

Comment:
should it be 'Release', instead of 'Debug'?

Reply:
This needs to be updated to match where the CI downloads the artifacts from the Windows build to. Current path is hardcoded from local testing.

Reply:
good catch, thanks! i believe this was because Scott was building with build_lib.bat rather than build.bat, updated now.